### DISCUSSION TITLE
When the cluster uses k3s, the ingress is configured, and the access to the resource route fails

### DISCUSSION THREAD

**Discussion by Zmzzz** [flet-dev/flet#3268] [Q&A]
![screenshot-20240509-133431](https://github.com/flet-dev/flet/assets/50945825/40a36865-efd4-48e4-8076-8ecef7f7fab4)
![image](https://github.com/flet-dev/flet/assets/50945825/e87ad419-0e45-411e-a45f-76a476649763)

**Maybe it's not a bug, but it's really bothering me a lot right now！**

When I used K3s ingress to configure the domain name, I needed to fill in a path to correspond to each container due to multiple containers. https://apic-xc599-vn.sczl.i3s.io/ in the figure above is the domain name, and /sczl/ points to a container. I have done redirection in the configuration to solve the routing problem of access, but when accessing static resources, flet only got the domain name, so the access could not be 404.

**Is there any configuration that allows me to modify the route to access static resources? I need to prefix the route with a /path/ prefix. The effect is as follows:**
![image](https://github.com/flet-dev/flet/assets/50945825/6153f61a-cf12-4460-a8b6-89789ce3f2a5)

**Comment by FeodorFitsner** [comment]
https://flet.dev/docs/publish/web/static-website#hosting-website-in-a-sub-directory
https://flet.dev/docs/publish/web/dynamic-website#environment-variables
